docs/pages/versions/unversioned/sdk/localization.md

```diff
@@ -54,27 +53,33 @@ Let's make our app support English and Japanese.
 <SnackInline label="Localization" dependencies={['expo-localization', 'i18n-js']}>
 
 ```tsx
-import * as React from 'react';
 import { View, StyleSheet, Text } from 'react-native';
 import * as Localization from 'expo-localization';
-import i18n from 'i18n-js';
+import { I18n } from 'i18n-js';
 
 // Set the key-value pairs for the different languages you want to support.
-i18n.translations = {
+const translations = {
   en: { welcome: 'Hello', name: 'Charlie' },
   ja: { welcome: 'こんにちは' },
 };
+const i18n = new I18n(translations);
+
 // Set the locale once at the beginning of your app.
 i18n.locale = Localization.locale;
+
 // When a value is missing from a language it'll fallback to another language with the key present.
-i18n.fallbacks = true;
+i18n.enableFallback = true;
+// To see the fallback mechanism uncomment line below to force app to use Japanese language.
+// i18n.locale = 'ja';
 
-export default App => {
+export default function App() {
   return (
     <View style={styles.container}>
       <Text style={styles.text}>
         {i18n.t('welcome')} {i18n.t('name')}
       </Text>
+      <Text>Current locale: {i18n.locale}</Text>
+      <Text>Device locale: {Localization.locale}</Text>
     </View>
   );
 };
```

Comment:
We can remove this semi-colon. The ESlint gives a warning on Snack: `Unnecessary semicolon. (no-extra-semi)
(ESLint)`.

Reply:
Hmm, that's weird, since this suggest the code parsing issue.

 I have formatted the code via Prettier on Snack, and errors like that came from the users-side ESLint config.

Where and when you see this warning, can you elaborate?

Reply:
I was seeing it when running the PR locally and opening the Snack. I saved the snack here: https://snack.expo.dev/@amanhimself/localization